python-backend/main.py

```diff
@@ -69,33 +54,63 @@ async def analyze(request: Request):
 
     try:
         data = Input(**(await request.json()))
-        (expression, a, b, symbol) = parse(data)
-        pcf = ramanujan.pcf.PCF(str(a), str(b))
```

Comment:
Due to a breaking change, this function is now unusable so it has been removed and replaced later with an approximation.